Gemfile

```diff
@@ -11,7 +11,8 @@ gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'foundation-rails'
 gem 'devise'
-
+gem "carrierwave"
+gem "fog"
 
 group :development, :test do
   gem 'byebug'
```

Comment:
Take a look at [this article](https://learn.launchacademy.com/lessons/rails-defaults) as to why you should remove the `byebug` gem.